crates/state/src/style.rs

```diff
@@ -36,6 +37,7 @@ use crate::{
 #[derive(Default, Debug, Clone, PartialEq, Component)]
 pub struct StyleState {
     pub background: Fill,
+    pub fill_stroke: (Option<Color>, Option<Color>),
```

Comment:
Why is it a tuple? better to just put each color in its own field right? Don't see the need for a tuple
```suggestion
    pub svg_fill: Option<Color>,
    pub svg_stroke: Option<Color>,
```